osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -54,6 +54,12 @@ public partial class BeatmapManager : DownloadableArchiveModelManager<BeatmapSet
         /// </summary>
         public IBindable<WeakReference<BeatmapInfo>> BeatmapRestored => beatmapRestored;
 
+        /// <summary>
+        /// A function which populates online information during the import process.
+        /// It is run as the final step of import.
+        /// </summary>
+        public Func<BeatmapSetInfo, CancellationToken, Task> PopulateOnlineInformation;
```

Comment:
would probably be good to have this marked `[CanBeNull]` for posterity at least

Reply:
Also noticed reviewing the next one in order that this is awkwardly placed between `BeatmapRestored` and its backing field.

Reply:
probably not worth fixing as these get moved later on in the chain.